src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/SNSConfiguration.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.sns.SnsAsyncClient
+import java.net.URI
+
+@Configuration
+class SNSConfiguration(
+  @Value("\${aws.region}") private val region: String,
+  @Value("\${aws.access-key-id}") private val accessKeyId: String,
+  @Value("\${aws.secret-access-key}") private val secretAccessKey: String,
+  @Value("\${aws.sns.uri}") private val uri: String,
```

Comment:
It is technically a URI but it's probably more colloquial to define an [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html).
What do you think about using `aws.sns.arn`?

🙋‍♂️ The env variable currently is `AWS_SNS_TOPIC_ARN` which doesn't seem to match up to this, unless I'm missing something.

Reply:
uri is different to arn. the uri is the endpoint i.e. localhost:5466 and arn is the topic definition. i'll rename uri to be clearer.